### ISSUE TITLE
TensorFlow Lite Post-training Integer Quantization predicts identical value always

### ISSUE THREAD

**Issue by stefano555** [tensorflow/tensorflow#40356]
**System information**
OS Linux 18.04.04 LTS
TensorFLow nightly 2.3.0-dev20200601
CPU Intel Core i7-855OU
ResNet-50 V2

Hi,

I have been experiencing issues converting ResNet-50 V2 from SavedModel to TensorFlow Lite Post-Training Integer Quantization. I have previously converted it to Tf Lite post-training dynamic range, post-training float16 quantization, and tf lite "normal". In all of those cases, it worked and performed as expected. Validation accuracy is 64.15%
I tried the float32 fallback option, the int8, and the uint8 option with identical issues. The conversion works fine but then when during test phase it predicts over and over again the same value.  Moreover, speed is 6s per inference whilst for the other versions is around 40 ms per inference. I also compiled the model for EdgeTPU. The conversion works but then again some issue, only one value is predicted for each image.

Please find below a link to a google folder with dataset, savedmodel, conversion code, and converted model. 

https://drive.google.com/drive/folders/11XruNeJzdIm9DTn7FnuIWYaSalqg2F0B?usp=sharing.

**Comment by sushreebarsa** [comment]
@stefano555 Could you please let us know if this is still an issue in TF v2.6.0 ? Thanks!
